week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/main.js

```diff
@@ -1,5 +1,6 @@
-import { root } from './shop/root.component.js'
-
-const $app = document.querySelector('#app')
-
-$app.append(root())
+"use strict";
+exports.__esModule = true;
+var root_component_js_1 = require("./shop/root.component.js");
+var $app = document.querySelector('#app');
+if ($app)
+    $app.append((0, root_component_js_1.root)());
```

Comment:
Skoro Twój `outDIr` to `build` to ten plik nie powinien być w repo - powinien być usunięty.